source/simulator/parameters.cc

```diff
@@ -142,6 +142,15 @@ namespace aspect
                        "Units: Years or seconds, depending on the ``Use years "
                        "in output instead of seconds'' parameter.");
 
+    prm.declare_entry ("Maximum relative increase in time step", boost::lexical_cast<std::string>(std::numeric_limits<int>::max()),
+                       Patterns::Double (0),
+                       "Set a percentage with which the the time step is limited to increase. Generally the "
+                       "time step based on the CFL number should be sufficient, but for complicated models "
+                       "which may suddenly drastically change behavoir, it may be usefull to limit the increase "
+                       "in the timestep, without limiting the time step size of the whole simulation to a "
+                       "particular number if this value is $50$, then that means that the time step can at most "
+                       "increase by 50\% from one time step to the next, or by a factor of 1.5. ");
```

Comment:
You'll need two backslashes here. The C++ preprocessor is going to convert the two into one in the string representation that is then given to LaTeX, and LaTeX needs one before the percent sign to escape it.

You also don't need the space at the end of the last line here, though that's a minor issue.

Reply:
you mean like what I did now?